assets/css/dogs-cats-birds-rabbits.css

```diff
@@ -3,49 +3,56 @@
     height:100vh;
     width:100%;
     display:flex;
-    background:grey;
-    position:relative;
-}
-.dogs-cats-birds-rabbits-wrapper::after{
-    position:absolute;
-    content:' ';
-    height:100%;
-    width:10%;
-    background:grey;
-    left:-10%;
-}
-.dogs-cats-birds-rabbits-wrapper::before{
-    position:absolute;
-    content:' ';
-    height:100%;
-    width:10%;
-    background:grey;
-    right:-10%;
+    background:#e5e5e5;
 }
-.inner-wrapper-small-margin{
+.inner-wrapper.dogs-cats-birds-rabbits{
     width:85%;
     margin:0 auto;
 }
 .dogs-cats-birds-rabbits-photo{
    width:40%;
    height:80%;
-   background:blue;
    position:relative;
    top:10%;
+   overflow: hidden;
 }
-.dogs-cats-birds-rabbits-rectangle-wrapper{
+.dogs-cats-birds-rabbits-box-wrapper{
     width:60%;
     height:100%;
     flex-wrap: wrap;
-    flex-direction: row;
     display:flex;
 }
-.dogs-cats-birds-rabbits-rectangle-up , .dogs-cats-birds-rabbits-rectangle-down{
+.dogs-cats-box , .birds-rabbits-box{
```

Comment:
Po co dwie klasy, skoro w środku mają dokładnie to samo?

Reply:
Poprawione